src/cpuload.c

```diff
@@ -297,7 +297,7 @@ int main(int argc, const char* argv[])
    const char *name;
    unsigned int load = 0;
 
-   printf ("\nCPU load generator, build %s %s.\n", __DATE__, __TIME__);
+   printf ("\nCPU load generator\n");
```

Comment:
You could replace __DATE__ and __TIME__with version which wouldn't change unless the package changes.

Reply:
Yeah but is it worth the money? We use RPM, so you rarely need to rely on build metadata embedded in the code. Hence I think it's better to simply drop such info.